### DISCUSSION TITLE
Generic Fandom -> Breezewiki redirect(s)

### DISCUSSION THREAD

**Discussion by marsdeat** [mshibanami/redirect-web#59] [Redirect Rules]
A generic rule that redirects all Wikia (Fandom) wiki URLs to a mirror (Breezewiki).

I recommend putting this near the bottom of your redirect rules list, as there are a lot of Wikis with independent alternatives that you may want to redirect more directly.

Redirect from: `https://*.fandom.com/*`
URL Pattern: wildcard
Redirect to: `https://breezewiki.com/$1/$2`
Example: `https://zelda.fandom.com/wiki/Zelda` becomes `https://breezewiki.com/zelda/wiki/Zelda`

There is also a second rule I use for when that mirror is under maintenance, to another mirror running the same software:

Redirect from: `https://*.fandom.com/*`
URL Pattern: wildcard
Redirect to: `https://antifandom.com/$1/$2`
Example: `https://zelda.fandom.com/wiki/Zelda` becomes `https://antifandom.com/zelda/wiki/Zelda`

I also have a LOT of specific Wiki redirects set up, but it would be overwhelming to list them all here.

**Comment by mshibanami** [comment]
Hi @marsdeat, this is great! Honestly, I misunderstood the power of this rule until now, but it’s actually a very useful rule!  

It's a bit late to say, but if you don’t mind, could I add your rule to the library? Thanks!

**Reply by marsdeat** [reply]
Absolutely, no problems from me! :)

**Reply by mshibanami** [reply]
Hmm, sorry, but I probably won’t add this rule for now, since BreezeWiki seems a bit unstable these days.

As far as I can tell, antifandom.com often returns this error [here](https://antifandom.com/matrix/wiki/Neo), for example:

> Sorry! Fandom isn't allowing BreezeWiki to show pages right now.

![Screenshot 2025-09-06 at 4 39 20 PM](https://github.com/user-attachments/assets/64cee16c-a6f1-43b4-b985-870b3b312122)

breezewiki.com also seems less reliable, as it often (or always?) returns 502.

On the other hand, I can see some [instances](https://github.com/KevinPayravi/indie-wiki-buddy/blob/0b89c5516aed53e4abf98acf668d9f472eb9f58b/manifest-firefox.json#L14-L32) working fine.

According to [the author](https://lists.sr.ht/~cadence/breezewiki-discuss/%3CCALHkHtKxOQ7-nNfmW-+A9FOoDf+FQGDFTHxFystUq3jFQMF-SA@mail.gmail.com%3E#%3C5e6d8767-ad49-42b5-871f-c346a5d7e548@disroot.org%3E), this is because their server has hit Fandom’s access limit.

I read BreezeWiki’s source code, and it seems to scrape Fandom pages with a user agent named “BreezeWiki,” without using multiple IP addresses or accounts to avoid being blocked, unlike Nitter.

This means that even if I add the rule to our library for an instance that currently works, the instance could quickly hit the access limit once the rule becomes popular.

Previously, I [introduced](https://github.com/mshibanami/redirect-web/issues/61#issuecomment-2713878372) a simple tool for auto-selecting instances called [jisk](https://github.com/mshibanami/jisk), and it could help avoid this issue. Maybe I’ll look into it later.

**Reply by alcayaga** [reply]
BreezeWiki changed how they access Fandom pages (looks like it loads the content in the browser) and now seems to work reliably
